docs/experiments.md

```diff
@@ -0,0 +1,30 @@
+# Experiments
+
+Fenix uses Mozilla's [Experiments service for experimentation](https://github.com/mozilla-mobile/android-components/blob/master/components/service/experiments/README.md).
+
+## Experiment Naming
+
+Naming must be unique and under 100 characters.
+
+Name experiments in the format of "fenix_experiment_issue", such as "fenix_etp_5651"
```

Comment:
Optional nit: this is already pretty clear, but what do you think about "fenix_{experiment}_{issue}"?